config/runtime.exs

```diff
@@ -0,0 +1,7 @@
+import Config
+
+config :set_game, :timeout, 60 * 60 * 24 * 1_000
+
+if config_env() == :test do
```

Comment:
You may want to break out environment specific things into its own config. See: https://github.com/DockYard/dockyard-api/blob/staging/config/config.exs#L92-L94

Reply:
Trying to do that I get:

```ex
** (CompileError) config/runtime.exs:9: invalid argument for import, expected a compile time atom or alias, got: "#{Mix.env()}.exs"
    (elixir 1.12.3) lib/code.ex:656: Code.eval_string_with_error_handling/3
    (elixir 1.12.3) lib/config.ex:258: Config.__eval__!/3
    (elixir 1.12.3) lib/config/reader.ex:86: Config.Reader.read!/2
```

Reply:
How are you running the code from the command line?